README.md

```diff
@@ -112,34 +135,239 @@ The Service Principal needs:
+**Key columns:**
+- `workspace_id` - ID of the workspace
+- `job_id` - ID of the job definition
+- `run_id` - ID of this specific run
+- `result_state` - Outcome (SUCCEEDED, FAILED, CANCELED, etc.)
+- `period_start_time` - Start time for this period
+- `period_end_time` - End time for this period
+- `run_type` - Type of run (JOB_RUN, WORKFLOW_RUN, etc.)
+
+### `system.lakeflow.job_task_run_timeline`
+
+Tracks individual task executions within jobs. Tasks can retry independently, making this useful for tracking retry behavior.
+
+**Key columns:**
+- `workspace_id` - ID of the workspace
+- `job_id` - ID of the parent job
+- `job_run_id` - ID of the parent job run
+- `run_id` - ID of this specific task run
+- `task_key` - Identifier for the task within the job
+- `result_state` - Outcome of the task
+- `period_start_time` - Start time
+- `period_end_time` - End time
+
+### `system.lakeflow.pipeline_update_timeline`
+
+Tracks Delta Live Tables (DLT) pipeline update executions. Records both successful and failed pipeline runs.
+
+> **⚠️ Permissions Note:** This table exists but may require explicit `SELECT` permission for your Service Principal. If the Service Principal lacks permission, you'll see "TABLE_OR_VIEW_NOT_FOUND" errors (even though the table exists). Grant the Service Principal `SELECT` permission on this table, and the exporter will automatically detect it and resume collection. See [Troubleshooting](#pipeline-metrics-not-available-table_or_view_not_found) for details.
+
+**Key columns:**
+- `workspace_id` - ID of the workspace
+- `pipeline_id` - ID of the pipeline
+- `update_id` - ID of this update
+- `request_id` - Request ID (multiple requests for same update indicate retries)
+- `result_state` - Outcome of the update
+- `period_start_time` - Start time
+- `period_end_time` - End time
+- `update_type` - Type of update (FULL_REFRESH, INCREMENTAL, etc.)
+
+### `system.query.history`
+
+Contains execution history for all SQL queries run in the workspace. Includes queries from SQL warehouses, notebooks, and serverless compute.
+
+**Key columns:**
+- `workspace_id` - ID of the workspace
+- `statement_id` - Unique identifier for the statement execution
+- `execution_status` - Status (FINISHED, FAILED, CANCELED)
+- `start_time` - When execution started
+- `end_time` - When execution finished
+- `total_duration_ms` - Total execution time in milliseconds
+- `error_message` - Error details if the query failed
+- `query_source` - Structured data about what triggered the query
+
 ## Metrics
 
-The exporter provides the following metrics:
+The exporter collects 18 metrics across four categories: billing, jobs, pipelines, and SQL queries.
+
+### Billing and Cost Metrics
+
+These metrics help with FinOps and cost tracking.
+
+#### `databricks_billing_dbus_total`
+Daily DBU consumption per workspace and SKU over the last 30 days.
+
+- **Source table:** `system.billing.usage`
+- **Type:** Counter
+- **Labels:** `workspace_id`, `sku_name`, `usage_date`
```

Comment:
I don't see `usage_date` added as a label anywhere in the codebase unless I'm missing something?

Reply:
I will address this point as part of improving the README.